mtags/src/main/scala-3/scala/meta/internal/pc/CompilerSearchVisitor.scala

```diff
@@ -20,8 +21,11 @@ class CompilerSearchVisitor(
 
   val logger: Logger = Logger.getLogger(classOf[CompilerSearchVisitor].getName)
 
+  private def isOwnerAccessible(sym: Symbol): Boolean =
+    val owner = sym.maybeOwner
+    owner.is(Flags.ModuleClass) || !(owner.isClass || owner.is(Flags.Trait))
```

Comment:
It's fine if the parent is an object but it may not be fine if the parent is an object and the grandparent is a class. If you just want to check if it is static I think there is a method for that on symbol. But a class owner can also be fine if it's in the correct scope. Don't we care about this here? Maybe you could try and use `sym.isAccessibleFrom`?

Reply:
I completely forgot that we can just check if a symbol is static and it works, thanks! 

>But a class owner can also be fine if it's in the correct scope. Don't we care about this here?

That’s a good point, but here we provide only completions for symbols that are out of scope, so I don't think there will ever be such a possibility that class owner is valid.